### PR TITLE
Add description fields to NetworkAttachmentDefinition CRD

### PR DESCRIPTION
**Pull request by dougbtv**

### CODE REVIEW

File: images/multus-daemonset-crio-pre1.16.yml [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
-  validation:
-    openAPIV3Schema:
-      properties:
-        spec:
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+	openAPIV3Schema:
```
Comment (s1061123) [inline images/multus-daemonset-crio-pre1.16.yml:20]: Could you please replace tab to space for the indent?

File: images/multus-daemonset-crio.yml [hunk 2]
```diff
@@ -17,13 +17,18 @@ spec:
       served: true
       storage: true
       schema:
-        openAPIV3Schema:
+	openAPIV3Schema:
```
Comment (s1061123) [inline images/multus-daemonset-crio.yml:20]: Could you please replace tab to space for the indent?

File: images/multus-daemonset-pre-1.16.yml [hunk 3]
```diff
@@ -1,25 +1,35 @@
...
-  validation:
-    openAPIV3Schema:
-      properties:
-        spec:
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+	openAPIV3Schema:
```
Comment (s1061123) [inline images/multus-daemonset-pre-1.16.yml:20]: Could you please replace tab to space for the indent?

**Comment by dougbtv** [comment]
Thanks for the eyes on the tabs! New commit up to address it.

**Pull request merged**